language/resource-api/README.md

```diff
@@ -408,13 +420,13 @@ class Puppet::Provider::AptKey::AptKey
     @apt_key_cmd.run(context, 'del', key_id)
 ```
 
-If the command is not available, a `Puppet::ResourceApi::CommandNotFoundError` will be raised. This can be easily used to fail the resources for a specific run, if the requirements for the provider are not yet met.
+If the command is not available, a `Puppet::ResourceApi::CommandNotFoundError` will appear. This can be used to fail the resources for a specific run if the requirements for the provider are not met.
```

Comment:
"raised" is a technical term in this case. as in "raise an exception", similar to "throw" in other programming languages.